### ISSUE TITLE
Updated Dates: Search Results Cards + Versioning 

### ISSUE THREAD

**Issue by eaquigley** [IQSS/dataverse#2607]
Currently, results cards have this date appearing:

![screen shot 2015-09-29 at 11 24 18 am](https://cloud.githubusercontent.com/assets/5922904/10172422/43143c94-66ad-11e5-8479-cc6df5fa0657.png)

However, that date has no label, isn't the publication date, and doesn't match the date in the most recent version based on the versions table:

![screen shot 2015-09-29 at 11 55 04 am](https://cloud.githubusercontent.com/assets/5922904/10172453/6c12036a-66ad-11e5-9b4a-37c3e7823c8a.png)

Having so many different dates is confusing so we need to rethink having the date on the result card be the one for major version changes (as it currently is) or be the most recently updated date (so including minor versions) as well as a label that says, "Last Updated:" so there is context as to what this date is.

**Comment by djbrooke** [comment]
Talked with @eaquigley about this inconsistency. Will be prioritized for a future release.

**Comment by mheppler** [comment]
This issue was also reported in issue #2061 by @suenjedt.

>I am going through scholar's dataverses and am a little confused about the dates shown in the cards. In particular in the screenshot below where dates dont match when comparing 3.6 and 4.0. Also I could not find the date Sept 30 2012 in any of the metadata fields.

![screen shot 2015-04-21 at 11 20 19](https://cloud.githubusercontent.com/assets/7300319/7255709/019a35ec-e819-11e4-8eaf-f207175b501a.png)
![screen shot 2015-04-21 at 11 18 02](https://cloud.githubusercontent.com/assets/7300319/7255710/019b87c6-e819-11e4-9edd-a5bd9ebf87d6.png)

**Comment by pdurbin** [comment]
The existing behavior is a result of a design decision made at an April 2014 meeting: https://docs.google.com/a/harvard.edu/document/d/1DWsEqT8KfheKZmMB3n_VhJpl9nIxiUjai_AIQPAjiyA/edit?usp=sharing

The date in question that appears on the card (the date that you're sorting by) is this: 'it will either be "Release Date" (which is the "Major Version Release Date") or "Creation Date"'

Here are the definitions used:

- Creation Date: the date the object was created in the database
- Publication Date: the date of first release, version 1, from draft to public
- Major Version Release Date: the date for the major version of a dataset
- Minor Version Release Date: the date for the minor version of a dataset

I'm fine with revisiting this decision, of course, but it would be useful to review the greater context by way of those meeting notes.

**Comment by pdurbin** [comment]
> doesn't match the date in the most recent version based on the versions table

In the screenshot above the versions table has no header but a header was added since then and now it's a bit confusing that a pre 1.0 draft shows a published date under the "Versions" tab. It's 2018 and the UI says I published this dataset in 2016 but I never did:

<img width="1348" alt="screen shot 2017-11-03 at 12 28 41 pm" src="https://user-images.githubusercontent.com/21006/32385029-3016471c-c093-11e7-9b74-037bf9faa6ea.png">

I stumbled upon this because over at https://help.hmdc.harvard.edu/Ticket/Display.html?id=254989#txn-5263138 I was fretting that a dataset was half published. I hope this means it's fine. 😄
